lib/sheldon/helpers.rb

```diff
@@ -0,0 +1,28 @@
+module Helpers
+
+  def locate_brain
+    relative_path = '~/sheldon2'
+    # relative_path = ENV['SHELDON_DATA_DIR'] || '~/sheldon'
+    Pathname(relative_path).expand_path # Deals with the use of ~ when referencing home directories
+  end
+
+  def load_db
+    YAML::Store.new(File.join(locate_brain, 'db.yaml'))
+  end
+
+  def remove_home(path)
+    home_path = Pathname(File.expand_path('~'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.